src/Surfnet/ServiceProviderDashboard/Infrastructure/DashboardBundle/Resources/views/EntityActions/actionsForDetail.html.twig

```diff
@@ -2,6 +2,6 @@
     {% include "@Dashboard/EntityActions/overviewAction.html.twig" %}
     {% include "@Dashboard/EntityActions/editAction.html.twig" with {action: entity} %}
     {% include "@Dashboard/EntityActions/secretResetAction.html.twig" with {action: entity} %}
-    {% include "@Dashboard/EntityActions/aclAction.html.twig" with {action: entity} %}
+    {% include "@Dashboard/EntityActions/aclAction.html.twig" with {action: en tity} %}
```

Comment:
Again with the profanity!

Reply:
😂 

Although in my defence I should point out that it's not profane to talk about a tity (as in a small tithe).  That'd be tittie you're thinking about 😇 

Fixed it!